1. Mexiko Create filenames filename-mappings and infofiles.py

```diff
@@ -326,13 +326,15 @@ def create_infofiles(row, filenames_file, not_ok_file):
             personnamn_not_even = True
             infotext += "|depicted people     = " + row["Personnamn / avbildad"] + "\n"
         else:
+            if "Linné, Sigvald" in row["Personnamn / avbildad"]:
```

Comment:
To clarify. What I meant was replacing everything from (the current) line 327 downwards by:
```
         else:
             for i, j in zip(lista[::2], lista[1::2]):
                if j + " " + i == ("Sigvald Linné", "Sigvald Linne"):
                    depicted_people += "[[:d:Q5959424|Sigvald Linné]]/"
                    content_categories_string += "[[Category:Sigvald_Linné]]\n"
                else:
                    depicted_people += j + " " + i + "/"
```
That is what you had originally was good apart from the accent and the linne_category both of which are fixed in the above code.